client/my-sites/customer-home/cards/actions/wp-for-teams-quick-links/index.jsx

```diff
@@ -44,74 +41,45 @@ export const QuickLinks = ( {
 
 	const quickLinks = (
 		<div className="wp-for-teams-quick-links__boxes quick-links__boxes">
-			{ isStaticHomePage ? (
-				<ActionBox
-					href={ editHomePageUrl }
-					hideLinkIndicator
-					onClick={ trackEditHomepageAction }
-					label={ translate( 'Edit homepage' ) }
-					materialIcon="laptop"
-				/>
-			) : (
-				<ActionBox
-					href={ `/post/${ siteSlug }` }
-					hideLinkIndicator
-					onClick={ trackWritePostAction }
-					label={ translate( 'Write blog post' ) }
-					materialIcon="edit"
-				/>
-			) }
-			{ isStaticHomePage ? (
-				<ActionBox
-					href={ `/page/${ siteSlug }` }
-					hideLinkIndicator
-					onClick={ trackAddPageAction }
-					label={ translate( 'Add a page' ) }
-					materialIcon="insert_drive_file"
-				/>
-			) : (
-				<ActionBox
-					href={ `/comments/${ siteSlug }` }
-					hideLinkIndicator
-					onClick={ trackManageCommentsAction }
-					label={ translate( 'Manage comments' ) }
-					materialIcon="mode_comment"
-				/>
-			) }
-			{ isStaticHomePage ? (
-				<ActionBox
-					href={ `/post/${ siteSlug }` }
-					hideLinkIndicator
-					onClick={ trackWritePostAction }
-					label={ translate( 'Write blog post' ) }
-					materialIcon="edit"
-				/>
-			) : (
-				<ActionBox
-					href={ `/page/${ siteSlug }` }
-					hideLinkIndicator
-					onClick={ trackAddPageAction }
-					label={ translate( 'Add a page' ) }
-					materialIcon="insert_drive_file"
-				/>
+			{ isP2Hub && (
+				<>
+					<ActionBox
+						href={ `/people/new/${ siteSlug }` }
+						hideLinkIndicator
+						onClick={ trackAddP2UsersAction }
+						label={ translate( 'Invite people to this workspace' ) }
+						materialIcon="person"
+					/>
+				</>
 			) }
-			{ showCustomizer && (
-				<ActionBox
-					href={ menusUrl }
-					hideLinkIndicator
-					onClick={ trackEditMenusAction }
-					label={ translate( 'Edit menus' ) }
-					materialIcon="list"
-				/>
-			) }
-			{ showCustomizer && (
-				<ActionBox
-					href={ customizeUrl }
-					hideLinkIndicator
-					onClick={ trackCustomizeThemeAction }
-					label={ translate( 'Customize theme' ) }
-					materialIcon="palette"
-				/>
+			{ ! isP2Hub && (
+				<>
+					<ActionBox
+						href={ `/post/${ siteSlug }` }
+						hideLinkIndicator
+						onClick={ trackWritePostAction }
+						label={ translate( 'Write post' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **44** times:
`translate( 'Write a Post' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/p2-quicklink-content)